scripts/get_mendeley_papers.py

```diff
@@ -0,0 +1,140 @@
+###
+# Copyright 2015-2020, Institute for Systems Biology
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+from __future__ import print_function
+
+from mendeley import Mendeley
+import json
+
+MENDELEY_PAPERS_FILE = "mendeley_papers.json"
+
+def test():
+    client_id = 9526
+    client_secret = "AmIvWP7FRxeLHX7n"
```

Comment:
Security risk! I would have the client_id and client_secret stored in a file, inside a gcs bucket that is not a public bucket. No security information should be stored in the repos. Also must change the client secret at this point.

Reply:
Thanks for catching that! This a test project I made in my personal Mendeley account. If I add them to private gcs bucket, does it mean I need to run the script in a GCP VM? And need to run the script in GCP VM for both dev and prod? Right now I am running locally and uploading the result JSON files. I can also remove these two info and ask the user to type them in.